lib/index.rb

```diff
@@ -59,13 +51,9 @@ def real_name
       # { real_name => { "aliases" => { alias => {} } } }
       # If not, ES would return {} before version 0.90, but raises a 404 with version 0.90+
       begin
-        alias_info = JSON.parse(@client.get("_aliases"))
-      rescue RestClient::ResourceNotFound => e
-        response_body = JSON.parse(e.http_body)
-        if response_body['error'].start_with?("IndexMissingException")
-          return nil
-        end
-        raise
+        alias_info = @client.indices.get_aliases(index: @index_name)
+      rescue Elasticsearch::Transport::Transport::Errors::NotFound
+        return nil
```

Comment:
For review: is it possible for `get_aliases` to 404 with an error besides `IndexMissingException, and if so do we care about it?